openmetadata-ui/src/main/resources/ui/src/components/Settings/Services/Services.tsx

```diff
@@ -206,12 +206,19 @@ const Services = ({ serviceName }: ServicesProps) => {
           filters: serviceTypeFilter?.length
             ? `(${serviceTypeFilter
                 .map((type) => `serviceType:${type}`)
-                .join(' ')})`
+                .join(' OR ')})`
             : undefined,
         });
       } else if (cursorType) {
         handlePageChange(currentPage);
-        getServiceDetails({ [cursorType]: paging[cursorType] });
+        getServiceDetails({
+          [cursorType]: paging[cursorType],
+          filters: serviceTypeFilter?.length
+            ? `(${serviceTypeFilter
+                .map((type) => `serviceType:${type}`)
+                .join(' OR ')})`
+            : undefined,
+        });
```

Comment:
This piece of code `filter` is repeated multiple times in this file, can we centralize it?